### PR TITLE
Minor Changes to avoid redundancy in code and follow code patterns

### PR DESCRIPTION
**Pull request by jgarciaf106**
@JCZuurmond 
I made the changes as per your request.

### CODE REVIEW

File: src/databricks/labs/lsql/dashboards.py [hunk 1]
```diff
@@ -897,32 +897,32 @@ def _from_dashboard_folder(cls, folder: Path) -> "DashboardMetadata":
             tiles.append(tile)
         return cls(display_name=folder.name, _tiles=tiles)
 
-    def export_to_zipped_csv(self, sql_backend: SqlBackend, export_path: Path) -> Path:
+    def export_to_zipped_csv(self, sql_backend: SqlBackend, export_path: Path, zip_file_name: str) -> Path:
```
Comment (JCZuurmond) [inline src/databricks/labs/lsql/dashboards.py:900]:
```suggestion
    def export_to_zipped_csv(self, sql_backend: SqlBackend, export_path: Path) -> Path:
```
Expect the zip file to be part of the export path

File: src/databricks/labs/lsql/dashboards.py [hunk 2]
```diff
@@ -897,32 +897,32 @@ def _from_dashboard_folder(cls, folder: Path) -> "DashboardMetadata":
...
         """Export the dashboard queries to CSV files directly into a ZIP archive."""
-        zip_export = export_path / "export_to_zipped_csv.zip"
+        zip_export = export_path / f"{zip_file_name}.zip"
 
         with ZipFile(zip_export, mode="w") as zip_file:
             for tile in self.tiles:
-                if tile.metadata.is_query():
-                    rows = sql_backend.fetch(tile.content)
+                if not tile.metadata.is_query():
+                    continue
```
Comment (JCZuurmond) [inline src/databricks/labs/lsql/dashboards.py:908]: Nit: remove a couple of the whitelines of code blocks logically belonging to gether
```suggestion
```

**Review by JCZuurmond — CHANGES_REQUESTED** [review]
Could you update the signature?

File: src/databricks/labs/lsql/dashboards.py (hunk 1 above)
Comment (jgarciaf106) [reply src/databricks/labs/lsql/dashboards.py:900]: @JCZuurmond 

Are you okay with 
        `if not export_path.suffix == '.zip':
            raise ValueError("The export_path must point to a .zip file.")`
To capture any path not pointing to a .zip path?

Comment (jgarciaf106) [reply src/databricks/labs/lsql/dashboards.py:900]: Or 
`        if not export_path.suffix == '.zip':
            logger.error("The export_path must point to a .zip file.")`

Comment (jgarciaf106) [reply src/databricks/labs/lsql/dashboards.py]: @JCZuurmond 
Removed extra  suffix validation on the export_to_zipped_csv function.

**Review by JCZuurmond — APPROVED** [review]
lgtm

**Pull request merged**